src/main/java/com/google/firebase/auth/UserRecord.java

```diff
@@ -177,6 +194,16 @@ public UserMetadata getUserMetadata() {
   }
 
   /**
+   * Returns additional custom claims set on this user.
```

Comment:
I don't understand what additional means in this context. I suppose it returns all the custom claims?

Reply:
Removed `additional`